modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+With this update, this behavior has been corrected. Selecting a string and running `tinymce.activeEditor.execCommand('insertParagraph');` now replaces the selection with a paragraph break, as expected.
+
+
+=== Triple-clicking did not select the paragraph in Google Chrome in some circumstances
+
+Previously, when triple-clicking a text paragraph in a {productname} document that —
+
+* included a nested `contenteditable` structure below the paragraph;
+
+* had an already-selected paragraph above
+
+— the paragraph was not selected.
+
+Instead the insertion point moved to the beginning of the paragraph. This only occurred if the host browser was Google Chrome. When the host browser was, for example, Safari or Firefox, triple-clicking a paragraph in this circumstance selected the paragraph as expected.
+
+With this update, triple-clicking text a paragraph above a `contenteditable` structure and below an already selected paragraph selects the clicked-on paragraph, no matter the host browser.
+
+NOTE: this update works around a Google Chrome bug by implementing on-triple-click selection behavior within {productname} regardless of the host browser.
+
+
+=== When text within an inline boundary element was selected and the right-arrow key was pressed, the insertion point incorrectly moved to the left
+
+{productname} provides an `inline_boundaries_selector` option. If this option is loaded, by default, it sets `a[href]`,`code`, and `.mce-annotation` as the elements inline boundaries apply to. Other inline elements — such as `strong`, `em` and `span` — can be added to these defaults.
+
+Previously, however, when a custom `inline_boundaries_selector` was initialised in a {productname} document, an erroneous keyboard navigation behavior presented.
+
+For example, if the following was added to `tinymce.init` —
+
+[source,js]
+===
+tinymce.init({
+  selector: 'textarea',
+  inline_boundaries_selector: 'span'
+});
+===
+
+— and text set within `<span></span>` tags was selected, pressing the right arrow key caused the text to be de-selected (expected behavior) and the insertion point to present to the left of the previously selected text (entirely not expected behavior).
+
+{productname} 6.1 corrects this. In the same circumstances as above, pressing the right arrow key in {productname} 6.1 puts the insertion point to the *right* of the newly de-selected text, as expected.
+
+
+=== Indenting or outdenting list items inside a block element that was inside another list item did not work
+
+If a list element within a list was also inside a block element, such as a <div></div>, placing the insertion point within any of the displayed lists and pressing *Tab* or *Shift-Tab* did not act as expected.
+
+Instead of indenting the list element (if *Tab* was pressed) or outdenting the list element (if *Shift-Tab* was pressed), nothing happened.
```

Comment:
```suggestion
Instead of indenting the list element (if *Tab* was pressed) or outdenting the list element (if *Shift+Tab* was pressed), nothing happened.
```